### DISCUSSION TITLE
Issue outputting a specified channel with 4a_compensate image data

### DISCUSSION THREAD

**Discussion by akandrew332** [angelolab/toffy#335] [Q&A]
I'm having trouble getting journal 4a_compensate_image_data to output tiffs of only the channels I want adjusted.  This works well using rosetta testing cell (44) by changing output_channel_names=None to output_channel_names='channels of interest'.  However, when moving down to cell 114 or Rosetta- Compensate Your Runs, I insert the bolded line with dsDNA as the channel of interest:
rosetta.compensate_image_data(raw_data_dir=os.path.join(extracted_imgs_dir, run), 
                                  comp_data_dir=os.path.join(rosetta_image_dir, run), 
                                  comp_mat_path=final_rosetta_path, panel_info=panel, **output_masses= 'dsDNA'**, batch_size=1)

I expected the same result, but instead I get a ValueError:
**ValueError: Not all values given in list output masses were found in list compensation masses.
 Displaying 1 of 1 invalid value(s) for list output masses
1            dsDNA**

Image of error attached.  I'm probably doing something wrong, or maybe this option isn't available for rosetta.compensate_image_data?  Any help would be greatly appreciated. Thanks!
![CompIssueWhenOutputting1Channel](https://user-images.githubusercontent.com/19846608/230141064-529f404b-3d95-4055-a0d0-18b1f5da786d.JPG)

**Comment by akandrew332** [comment]
Okay, So I've figured out that I need to put in the mass of the channel I'm interested in here rather than the channel name.  However, now I get the following error:
733: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  mask |= (ar1 == a)

**Reply by srivarra** [reply]
Could you try putting the output masses in a list, such as `[mass]` as the parameter.

**Reply by akandrew332** [reply]
Yup, that did it.  Thanks!
